libqtile/backend/wayland/core.py

```diff
@@ -166,6 +167,8 @@ def __init__(self):
             output_power_manager.set_mode_event, self._on_output_power_manager_set_mode
         )
         self.idle = Idle(self.display)
+        _idle_ihibitor_manager = IdleInhibitorManagerV1(self.display)
+        self.add_listener(_idle_ihibitor_manager.new_inhibitor_event, self._on_new_inhibitor)
```

Comment:
` _idle_ihibitor_manager` -> ` idle_ihibitor_manager` just to be consistent with other local-only variables